# HG changeset patch
# User andrew
# Date 1424102734 0
#      Mon Feb 16 16:05:34 2015 +0000
# Node ID 4fdaf786d977aa77afdb68b8829579d31069e39c
# Parent  01a0011cc101f3308c5876db8282c0fc5e3ba2e6
PR2236: ppc64le should report its os.arch as ppc64le so tools can detect it
Summary: Use ppc64le as the arch directory on that platform and report it in os.arch

--- openjdk/hotspot/make/defs.make.orig	2015-03-03 17:05:51.000000000 -0500
+++ openjdk/hotspot/make/defs.make	2015-03-03 19:18:20.000000000 -0500
@@ -319,6 +319,13 @@
   LIBARCH/ppc64   = ppc64
   LIBARCH/zero    = $(ZERO_LIBARCH)
 
+  # Override LIBARCH for ppc64le
+  ifeq ($(ARCH), ppc64)
+    ifeq ($(OPENJDK_TARGET_CPU_ENDIAN), little)
+      LIBARCH = ppc64le
+    endif
+  endif
+
   LP64_ARCH += sparcv9 amd64 ia64 ppc64 aarch64 zero
 endif
 
